src/test/java/org/mule/extension/db/integration/update/BulkUpdateTestCase.java

```diff
@@ -61,6 +66,12 @@ public void updateBulkAfterSelect() throws Exception {
     assertBulkUpdate(response);
   }
 
+  @Test
+  public void updateBulkAfterSelectThrowsSqlException() throws Exception {
+    exceptionRule.expect(Exception.class);
```

Comment:
I was not able to use an SQLException here because this is sometimes wrapped in a MessagingException for some database engines